### ISSUE TITLE
Low performance

### ISSUE THREAD

**Issue by wangchuan12138** [AnTao97/UnsupervisedPointCloudFeatureLearning#2]
hi thanks for giving the source code, I am very interested in the project ,but i found some problem in reimplement. After the training for 300 epochs  I test the reconstruction performance , the base condition are  k=16 shape=plane encoder=foldnet,batch_size=4, dataset=shapenetcorev2, but the cd loss is 36, too high ...  And i download the pretrained model in same condition the loss is 16, It is quite different from the data in the paper. I want to ask you for some details , maybe i forgot something... thanks for reading

**Comment by antao97** [comment]
See README:

`Other than using the modified Chamfer distance in FoldingNet paper, we adopt the original Chamfer distance proposed by A Point Set Generation Network for 3D Object Reconstruction from a Single Image:`

**Comment by wangchuan12138** [comment]
you mean the codes i get from the github is not the final version?

**Comment by wangchuan12138** [comment]
the cd distance compute methods in this project need to change? i try to train the reconstruct model for 500 epochs on shapenet , i evaluate the model on modelnet the cd loss is still 22.23, a bit different to the paper.

**Comment by wangchuan12138** [comment]
May I have your email address I can send some details to you ? Thanks for giving attention.

**Comment by antao97** [comment]
My email is ta19@mails.tsinghua.edu.cn.

The CD distance I use in this repo is slightly different from the FoldingNet paper's CD distance. When you test the trained model with CD distance, you need to modify the CD distance to the FoldingNet style.
